CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 2.8.12)
```

Comment:
OK, dropped the `INTERFACE`.

@mattyclarkson I don't know cmake well enough to confidently pull that off, and it seems like unnecessary complexity.